src/app/layout.tsx

```diff
@@ -0,0 +1,79 @@
+import type { Metadata } from "next";
+import { Poppins } from "next/font/google";
+import "./globals.scss";
+import styles from './layout.module.scss';
+import { ReactNode } from "react";
+import Image from 'next/image';
+
+const poppins = Poppins({
+  variable: "--font-poppins",
+  subsets: ["latin"],
+  weight: ["400", "700"],
+});
+
+export const metadata: Metadata = {
+  title: "openSUSE ­— Open source OS for desktops, servers and containers",
+  description: "openSUSE is an opensource operating system based on Linux.",
```

Comment:
Unify spelling of "open source".

Reply:
openSUSE is also not an operating system :D